spec/controllers/issues_controller_spec.rb

```diff
@@ -0,0 +1,92 @@
+require "rails_helper"
+RSpec.describe IssuesController, type: :controller do
+  describe "#index" do
+    it "renders the index template" do
+      get :index
+      expect(response).to render_template("index")
+    end
+
+    context "without any additional params" do
+      let!(:issue1) { create(:issue) }
+      let!(:issue2) { create(:issue) }
```

Comment:
Tiny linting nitpick - we probably want a new line here before the it block

Reply:
Added a new line before the it block as suggested!